roles/StackStorm.st2/defaults/main.yml

```diff
@@ -32,6 +32,36 @@ st2_auth_enable: yes
 st2_auth_username: testu
 # Password used by StackStorm standalone authentication
 st2_auth_password: testp
+
+
+# By specifying a valid configuration for LDAP,
+# (See https://docs.stackstorm.com/latest/authentication.html#ldap )
+# LDAP auth backend is setup for st2.
+# Note that you just need to provide the backend_kwargs.
+st2_ldap_enable: no
+# "enable" is not a key of st2_ldap because the defaults would be lost if any key in the dictionary is changed
```

Comment:
:+1:

Reply:
Oh, you were right regarding `st2-apply-rbac-definition`. I added it now and added a task to set the rbac.backend to default (this prevented st2api from starting with rbac enabled).